its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ParameterTest.java

```diff
@@ -40,7 +40,6 @@
 
 @ExtendWith({ServerTests.class, ContextExtension.class})
 class ParameterTest {
-  private static final String SONAR_RULES_PREFIX = "csharpsquid:";
```

Comment:
Some of these are gone. While splitting the big test class, I was just copy/pasting them where needed. Most of them do not need to exist.